source/_integrations/zwave_me.markdown

```diff
@@ -34,6 +34,28 @@ This integration allows you to control a Z-Wave network via the [Z-Wave.Me Z-Way
 
 {% include integrations/config_flow.md %}
 
+In the URL field input the IP address with the port. The IP address can be prefixed with wss:// if HTTPS should be used instead of HTTP (when using find.z-wave.me remote access or public IP with SSL).
```

Comment:
Same comment as the original PR.

Please use `configuration_basic` blocks here to describe fields.

Thanks 👍 

../Frenck